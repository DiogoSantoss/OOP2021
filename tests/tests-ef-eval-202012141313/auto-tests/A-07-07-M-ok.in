8
2
R2
1
3O3
1
2
R2
1
7M5
1
2
R2
2
8B1256
1
2
R2
2
1C6H5OH
2
2
R2
2
1C6H5OH
4
0
6
5
R2
0
8
4
1
4
2
4
3
0
6
5
S2
5
R2
0
5
5
3O3
100
5
7M5
25
5
1C6H5OH
10
0
6
5
R2
0
0
