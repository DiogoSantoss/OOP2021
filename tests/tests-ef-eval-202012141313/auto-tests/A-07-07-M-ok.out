Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: 0|R2|3O3|1|8|8|1
1|R2|7M5|1|8|8|1
2|R2|8B1256|1|58|58|2
3|R2|1C6H5OH|2|4|4|2
4|R2|1C6H5OH|4|8|8|2
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Identificador da transacção: Gestão de Transacções
1 - Visualizar
2 - Registar venda
3 - Registar encomenda
4 - Pagar
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: 0|R2|3O3|1|8|8|1
1|R2|7M5|1|8|8|1|0
2|R2|8B1256|1|58|58|2|0
3|R2|1C6H5OH|2|4|4|2|0
4|R2|1C6H5OH|4|8|8|2
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Identificador do produto: Preço: Gestão de Produtos
1 - Visualizar
2 - Registar Caixa
3 - Registar Contentor
4 - Registar Livro
5 - Alterar Preço
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Identificador do cliente: 0|R2|3O3|1|8|8|1
1|R2|7M5|1|8|8|1|0
2|R2|8B1256|1|58|58|2|0
3|R2|1C6H5OH|2|4|4|2|0
4|R2|1C6H5OH|4|8|8|2
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 